service/org.nemomobile.BtrfsBalancer.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.nemomobile.BtrfsBalancer
+Name=org.nemomobile.BtrfsBalancer
+Exec=/usr/sbin/btrfs-balancer server
+User=root
```

Comment:
Adding the line is no problem. The earliest it could be called is either when the user invokes the OS update UI, or when the weekly auto-balancing gets triggered by its systemd timer, whichever comes first.